src/instr/instr_paje_trace.cpp

```diff
@@ -331,26 +309,18 @@ void DefineEventTypeEvent::print() {
   }
 }
 
-DefineLinkTypeEvent::DefineLinkTypeEvent(type_t type, type_t source, type_t dest)
+void DefineLinkTypeEvent(type_t type, type_t source, type_t dest)
 {
-  this->event_type                         = PAJE_DefineLinkType;
-  this->timestamp                          = 0;
-  this->type   = type;
-  this->source = source;
-  this->dest   = dest;
+  e_event_type event_type                         = PAJE_DefineLinkType;
+  double timestamp                          = 0;
```

Comment:
Please remove this variable `timestamp`, and all other timestamp variables that are not used but in XBT_DEBUG calls.